draft-dekater-scion-dataplane.md

```diff
@@ -1374,6 +1374,11 @@ In addition, the MAC algorithm can be freely chosen by each AS, enabling algorit
 A more realistic risk to the secrecy of the forwarding key is exfiltration from a compromised router or control plane service.
 An AS can optionally rotate its forwarding key at regular intervals to limit the exposure after a temporary device compromise. However, as is perhaps self-evident, such a key rotation scheme cannot mitigate the impact of an undiscovered, permanent compromise of a device.
 
+When an AS's forwarding key is compromised, an attacker can forge hop field MACs, undermining path authorization. As path segments are only checked for validity and policy compliance during path discovery during normal operation, and routers only perform minimal sanity checks of hop fields with valid MACs, creating fraudulent hop fields with valid MACs allows an attacker to bypass most path segment validity checks. Thus, an attacker can create path segments that violate the AS's local policy and/or general path segment validity requirements.
```

Comment:
> As path segments are only checked for validity and policy compliance during path discovery during normal operation, and routers only perform minimal sanity checks of hop fields with valid MACs, creating fraudulent hop fields with valid MACs allows an attacker to bypass most path segment validity checks.

Speaking that one requires breathing exercises. Any way to split it? (yeah, me, the lumper :-) )

Reply:
Done, but I felt I had to combine the next sentences to make it work.
Better?

Reply:
Yes.